autoload/mundo.vim

```diff
@@ -409,12 +411,59 @@ function! s:MundoRefresh()"{{{
   call winrestview(winView)
 endfunction"}}}
 
+function! s:MundoUpdateCursor()"{{{
+    if !g:mundo_auto_preview
+        return s:MundoRefresh()
+    endif
+
+    if !get(g:, 'mundo_inline_undo_delay', 0)
+        call s:MundoRefresh()
+    endif
+
+    if get(g:, 'mundo_auto_preview_delay', 0) <= 0
+        return s:MundoPython('MundoRenderPreview()')
+    endif
+
+    call s:MundoRestartCursorTimer()
+endfunction"}}}
+
+function! s:MundoRestartCursorTimer()"{{{
+    call s:MundoStopCursorTimer()
+    let s:auto_preview_timer = timer_start(
+                \ get(g:, 'mundo_auto_preview_delay', 0),
+                \ function('s:MundoRenderPreviewDelayed'))
+endfunction"}}}
+
+function! s:MundoStopCursorTimer()"{{{
+    if s:auto_preview_timer != -1
+        call timer_stop(s:auto_preview_timer)
+        let s:auto_preview_timer = -1
+    endif
+endfunction"}}}
+
+function! s:MundoRenderPreviewDelayed(...)"{{{
+    if s:auto_preview_line == line('.')
+        return
+    endif
+
+    if mode() != 'n'
+        return s:MundoRestartCursorTimer()
+    endif
+
+    if get(g:, 'mundo_inline_undo_delay', 0)
+        call s:MundoRefresh()
+    endif
+
+    let s:auto_preview_line = line('.')
+    call s:MundoPython('MundoRenderPreview()')
+endfunction"}}}
+
 augroup MundoAug
     autocmd!
     autocmd BufNewFile __Mundo__ call s:MundoSettingsGraph()
     autocmd BufNewFile __Mundo_Preview__ call s:MundoSettingsPreview()
-    autocmd CursorHold * call s:MundoRefresh()
-    autocmd CursorMoved * call s:MundoRefresh()
+    autocmd CursorMoved __Mundo__ call s:MundoUpdateCursor()
+    autocmd BufLeave __Mundo__ call s:MundoStopCursorTimer()
```

Comment:
Restricting this autocmd to the __Mundo__ window means that users that leave the __Mundo__ window open and make changes, will not see their changes updated.

Reply:
(I think you'll want to use *)

Reply:
My bad, I should have picked this up while testing. Should be an easy fix.